src/feature/feature.jl

```diff
@@ -0,0 +1,25 @@
+export line, area
+
+line(len::Int; background=' ') = DisplayRow(len, background, fill(Char(0x2500), len))
+
+function area(h::Int, w::Int; background=' ', border=true)
+    array = DisplayArray(h, w, background=background)
+    if border
+
+        # top, bottom ─
+        array.context[1] = line(w, background=background)
+        array.context[end] = line(w, background=background)
+
+        # side │
+        for i=2:(h-1) array[i, 1] = Char(0x2502) end
+        for i=2:(h-1) array[i, end] = Char(0x2502) end
```

Comment:
can be replaced with `fill!` and `view`

Reply:
I didn't implement `getindex` for `DisplayArray`

Reply:
then try `foreach`